scipy/stats/_binned_statistic.py

```diff
@@ -204,9 +206,9 @@ def binned_statistic_2d(x, y, values, statistic='mean',
     -------
     statistic : (nx, ny) ndarray
         The values of the selected statistic in each two-dimensional bin
-    xedges : (nx + 1) ndarray
+    x_edge : (nx + 1) ndarray
```

Comment:
I like the underscore, but these return values do contain multiple edges. so add the ``s`` back at the end?